app/views/pages/medicaid/intro_household.erb

```diff
@@ -0,0 +1,33 @@
+<h4 class="step-header step-header__title--centered">Introduction</h4>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Now tell us about any other people in your household.</div>
+    <p class="text--help text--centered">
+      This includes everyone who lives in your home (including you), even if they are not seeking benefits today.
+    </p>
+  </header>
+
+  <div class="form-card__content">
+    <div class="slab slab--not-padded">
+      <div class="card card--narrow">
+        <p>
+          <i class="button__icon--left icon-check icon-check--color"></i>
+          Just me!
```

Comment:
will this change dynamically if there is more than one person? Just curious / thinking ...

Reply:
Yes, it will change to Member Name (Me!). If we want to avoid that dynamic change, I can update it with that text even for one person